integr8ly/integreatly-operator
# Move integreatly-operator manifests and manifest generation into this repo

Removed these manifests from the manifests repo, and put them into this repo. Along with a makefile directive to push them to an application registry.

## Verification
- Delete the integreatly-operator from your personal quay.io application registry
- Run `make push/csv QUAY_USERNAME=xxx QUAY_PASSWORD=xxx REPO=xxx`
- See the new application in your registry
- Create an operator source pointing at your registry
- Install integreatly-operator
- See it succesfully running.
